README.md

```diff
@@ -19,8 +19,8 @@ Be sure to add the job_scheduler crate to your `Cargo.toml`:
 tokio-cron-scheduler = "*"
 ```
 
-Creating a schedule for a job is done using the `FromStr` impl for the
-`Schedule` type of the [cron](https://github.com/zslayton/cron) library.
+Creating a schedule for a job is done using the `new` impl for the
+`Cron` type of the [croner](https://github.com/https://github.com/Hexagon/croner-rust/cron) library.
```

Comment:
```suggestion
`Cron` type of the [croner](https://github.com/Hexagon/croner-rust) library.
```

Reply:
Thanks Adrian, yeah I'm not sure what happened there!